pyproject.toml

```diff
@@ -78,7 +77,7 @@ version_scheme = "release-branch-semver"
 
 [tool.black]
 line-length = 79
-target-version = ['py38']
+target-version = ['py39']
```

Comment:
I have to admit that I have not fully understood this line
https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html

Reply:
@rcomer I've interpret it as the minimum compatible version that you want `black` to understand (as a hint) so that it can parse the code.

So this change seems reasonable to me 👍